sqlglot/dialects/doris.py

```diff
@@ -49,6 +49,7 @@ class Generator(MySQL.Generator):
             exp.ArrayUniqueAgg: rename_func("COLLECT_SET"),
             exp.CurrentTimestamp: lambda self, _: self.func("NOW"),
             exp.DateTrunc: lambda self, e: self.func("DATE_TRUNC", e.this, unit_to_str(e)),
+            exp.GroupConcat: lambda self,e: exp.func("GROUP_CONCAT", e.this, e.args.get("'seprator'") or exp.Literal.string(",")),
```

Comment:
this is wrong, and not tested, " 'separator' " has extra quotes. 

can you please fix this and double check your work next time? if these low effort prs continue, we will no longer accept future contributions from you.